src/layouts/layouts.css

```diff
@@ -90,6 +91,7 @@
 
 .layout-3-1 > div {
     flex: 1;
+    max-height: calc(96% / 2);
```

Comment:
Oh , i used the calc for both of them bcos i thought it would be easier to divide an odd number 😉